### ISSUE TITLE
Agent sessions: there should be a command and keybinding to focus the recent sessions view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#281330]
And we should include this in the chat accessiblity help dialog so that screen reader and keyboard only users can easily access it.
